src/components/Modals/FiatRamps/config.ts

```diff
@@ -47,6 +50,20 @@ export const supportedFiatRamps: SupportedFiatRamp = {
     },
     isImplemented: true,
   },
+  [FiatRamp.Banxa]: {
+    label: 'fiatRamps.banxa',
+    info: 'fiatRamps.banxaMessage',
+    logo: banxalogo,
+    isImplemented: true,
+    getBuyAndSellList: async () => {
+      const assets = getBanxaAssets()
+      return [assets, assets]
```

Comment:
Would be more declarative and easier to read using something like this
```suggestion
      const buyAssets = getBanxaAssets()
      const sellAssets = buyAssets
      return [buyAssets, sellAssets]
```